CONTRIBUTING.md

```diff
@@ -61,7 +61,12 @@ If you wish to build a copy of Babel for distribution, then run:
 $ make build-dist
 ```
 
-and access the files from `packages/babel-core/dist`.
+and access the built files for individual packages from `packages/<package-name>/lib`.
+
+The "top-level" package is `babel-cli`.  Accordingly, its built files live in `packages/babel-cli/lib`.
```

Comment:
Not sure what top-level is supposed to mean in this case. Many people use `babel-core` indirectly through other modules.

Also not sure this is the best place for this, since this isn't really `Setup` related and just explains how `babel-core` is organized.